common/types/string.go

```diff
@@ -221,3 +240,447 @@ func stringStartsWith(s String, pre ref.Val) ref.Val {
 	}
 	return Bool(strings.HasPrefix(string(s), string(preStr)))
 }
+
+func stringFormat(s String, args ref.Val) ref.Val {
```

Comment:
The format syntax needs to be documented _somewhere_.

cc @TristonianJones for ideas on where that should be.

Reply:
I'll add the docs to the string extension library documentation that already exists, which seems like a good starting point.

Reply:
(marking as resolved)